src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -2439,16 +2418,25 @@ private bool Fix(int iParam, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
             {
                 if (lower != null)
                 {
-                    candidates.AddAll(lower, conversions);
+                    if (!AddAllCandidates(candidates, lower, VarianceKind.Out, conversions))
```

Comment:
>AddAllCandidates [](start = 25, length = 16)

nit: It would be useful to comment on the association: lower -> out, upper -> in.